### DISCUSSION TITLE
Support for EVPN Multi-Homing (EVPN-MH)

### DISCUSSION THREAD

**Discussion by jbemmel** [ipspace/netlab#846] [Features]
See https://docs.frrouting.org/en/stable-8.0/bgp.html#evpn-multihoming for FRR

What do people prefer: 
1) New module (evpn-mh)
2) New plug-in or 
3) make it part of the existing EVPN module?

**Comment by ipspace** [comment]
Looks like all you need is the same ESI across multiple interfaces. If that's the case, then we can add **evpn.esi** link attribute to the EVPN module. Don't want to go down any compatibility rabbit trails though.
